core/src/main/resources/lib/hudson/iconSize.jelly

```diff
@@ -41,9 +41,9 @@ THE SOFTWARE.
       </d:taglib>
 
       <td xmlns:local="local">${%Icon}:
-        <local:iconSizeLink title="S" sz="16x16" />
-        <local:iconSizeLink title="M" sz="24x24" />
-        <local:iconSizeLink title="L" sz="32x32" />
+        <local:iconSizeLink title="${%S}" sz="16x16" />
+        <local:iconSizeLink title="${%M}" sz="24x24" />
+        <local:iconSizeLink title="${%L}" sz="32x32" />
```

Comment:
I recommend to make these %small/%medium/%large and have them localize in `iconSize.properties` to S/M/L. That would probably make for better localization files.

Reply:
Good idea.